scripts/deploy_s3.py

```diff
@@ -0,0 +1,73 @@
+import argparse
```

Comment:
nit: it might be nice to add a comment to explain what this does (deploy to s3 so that snippets can pull the latest vers)

Reply:
Didn't see this before merge. Will do in follow up PR